### ISSUE TITLE
Daylight savings time causing times to be off on set notifications by 1 hour.

### ISSUE THREAD

**Issue by kuanb** [slco-2016/ch-courtsms#317]
To resolve this, look at the `moment.js` documentation under zone name (`moment.fn.zoneName`) to parse time zone. It might also be a good strategy to do something client side to determine what time zone the browser is in and fallback to the organization time zone in the cases where no time zone is returned.

Reference for `moment.js` docs: http://momentjs.com/timezone/docs/

**Comment by tdooner** [comment]
OHHHH I don't think this is a bug -- it's just that SLCO is an hour ahead of us!

Most times that are displayed in the product seem to be run through `moment.tz` so I think this is probably good.

**Comment by tmaybe** [comment]
Is the app's time zone set to the Mountain Time Zone somewhere in the application or config?

**Comment by tdooner** [comment]
Yeah, in the application. Each organization has a time zone. For SLCO it is set to `America/Denver` which is the way to do it correctly to allow moment.js to compensate for DST. Also it appears that most timestamps are full datetimes instead of just times, which is another common pitfall I have, uh, personally experienced, when compensating for DST.

**Comment by tmaybe** [comment]
Reading the text of this issue more closely, it seems that it might be referring to a mismatch happening when a notification is created (presumably through the application GUI). Like, you set a notification for 4pm and it's actually scheduled an hour off. Not sure if it refers to notifications created when the browser's in a time zone other than Mountain, or ...?
